sdk/synapse/Azure.Analytics.Synapse.Artifacts/tests/SparkJobDefinitionClientLiveTests.cs

```diff
@@ -33,5 +69,82 @@ private SparkJobDefinitionClient CreateClient()
                 InstrumentClientOptions(new ArtifactsClientOptions())
             ));
         }
+
+        [Test]
+        public async Task TestGetSparkJob()
+        {
+            SparkJobDefinitionClient client = CreateClient ();
+            await using DisposableSparkJobDefinition sparkJobDefinition = await DisposableSparkJobDefinition.Create (client, Recording, TestEnvironment.StorageFileSystemName, TestEnvironment.StorageAccountName);
+
+            IList<SparkJobDefinitionResource> jobs = await client.GetSparkJobDefinitionsByWorkspaceAsync().ToListAsync();
+            Assert.GreaterOrEqual(jobs.Count, 1);
+
+            foreach (var expectedJob in jobs)
+            {
+                SparkJobDefinitionResource actualJob = await client.GetSparkJobDefinitionAsync(expectedJob.Name);
+                Assert.AreEqual(expectedJob.Name, actualJob.Name);
+                Assert.AreEqual(expectedJob.Id, actualJob.Id);
+            }
+        }
+
+        [Test]
+        public async Task TestDeleteSparkJob()
+        {
+            SparkJobDefinitionClient client = CreateClient();
+
+            SparkJobDefinitionResource resource = await DisposableSparkJobDefinition.CreateResource (client, Recording, TestEnvironment.StorageFileSystemName, TestEnvironment.StorageAccountName);
+
+            SparkJobDefinitionDeleteSparkJobDefinitionOperation deleteOperation = await client.StartDeleteSparkJobDefinitionAsync  (resource.Name);
+            Response response = await deleteOperation.WaitForCompletionAsync ();
+            switch (response.Status) {
+                case 200:
+                case 204:
+                    break;
+                default:
+                    Assert.Fail($"Unexpected status ${response.Status} returned");
+                    break;
+            }
```

Comment:
nit: consider factoring this out to a helper since it is repeated in a few places.

Reply:
Agreed. I'll do it in the next PR though (to keep them small).